src/sentry/rules/processing/delayed_processing.py

```diff
@@ -295,9 +295,7 @@ def get_group_to_groupevent(
 def process_delayed_alert_conditions(buffer: RedisBuffer) -> None:
     with metrics.timer("delayed_processing.process_all_conditions.duration"):
         fetch_time = datetime.now(tz=timezone.utc)
-        project_ids = buffer.get_sorted_set(
-            PROJECT_ID_BUFFER_LIST_KEY, min=0, max=fetch_time.timestamp()
-        )
+        project_ids = buffer.get_sorted_set(PROJECT_ID_BUFFER_LIST_KEY, min="-inf", max="+inf")
```

Comment:
```suggestion
        project_ids = buffer.get_sorted_set(PROJECT_ID_BUFFER_LIST_KEY, min=float("-inf"), max=float("+inf"))
```
I _believe_ this will fix the type checks for you and work? (idk, gpt helped me.. so we'll see.)